src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -517,16 +507,6 @@ def _update_stack_component(
             # handle any potential renamed component
             updated_component.name = component.name
 
-            # rename components inside stacks
```

Comment:
These are no longer needed if the component UUID is used as a foreign key instead of the name+type